pages/index.js

```diff
@@ -1,14 +1,16 @@
 import Head from 'next/head';
 import Image from 'next/image';
```

Comment:
import that you are not using, you can delete until you actually want to use them :)

Reply:
done:)